runtime/vm/visible.c

```diff
@@ -343,6 +355,11 @@ loadAndVerifyNestHost(J9VMThread *vmThread, J9Class *clazz, UDATA options)
 		/* If a problem occurred in nest host verification then the nest host value is invalid */
 		if (J9_VISIBILITY_ALLOWED == result) {
 			clazz->nestHost = nestHost;
+		} else if (J2SE_VERSION(vmThread->javaVM) >= J2SE_V15) {
+			clazz->nestHost = curClazz;
```

Comment:
Please add a comment here to reference the JVMS change that OKs this behaviour.  We want to explain that the exception is not rethrown if the nestHost is not accessible, it just means the class becomes it's own host.

See the Access Control section of the JDK15 JVMS:
>Otherwise,MhasaNestHostattribute,anditshost_class_indexitemisused as an index into the run-time constant pool of M. The symbolic reference at that index is resolved (§5.4.3.1).
If resolution of the symbolic reference fails, then M is its own nest host. Any exception thrown as a result of failure of class or interface resolution is not rethrown.

http://cr.openjdk.java.net/~iris/se/15/latestSpec/java-se-15-jvms-fr-diffs.pdf

Reply:
Comment added.